packages/clerk-js/src/ui/components/SignIn/ResetPassword.tsx

```diff
@@ -0,0 +1,114 @@
+import { useCallback } from 'react';
+
+import { clerkInvalidFAPIResponse } from '../../../core/errors';
+import { withRedirectToHomeSingleSessionGuard } from '../../common';
+import { useCoreSignIn } from '../../contexts';
+import { Col, descriptors, localizationKeys } from '../../customizables';
+import { Card, CardAlert, Form, Header, useCardState, withCardStateProvider } from '../../elements';
+import { useSupportEmail } from '../../hooks/useSupportEmail';
+import { useRouter } from '../../router';
+import { handleError, useFormControl } from '../../utils';
+
+export const _ResetPassword = () => {
+  const signIn = useCoreSignIn();
+  const card = useCardState();
+  const { navigate } = useRouter();
+  const supportEmail = useSupportEmail();
+
+  const passwordField = useFormControl('password', '', {
+    type: 'password',
+    label: localizationKeys('formFieldLabel__newPassword'),
+    isRequired: true,
+    enableErrorAfterBlur: true,
+    complexity: true,
+    strengthMeter: true,
+  });
+  const confirmField = useFormControl('confirmPassword', '', {
+    type: 'password',
+    label: localizationKeys('formFieldLabel__confirmPassword'),
+    isRequired: true,
+    enableErrorAfterBlur: true,
+  });
+
+  const canSubmit = passwordField.value.trim().length > 7 && passwordField.value === confirmField.value;
```

Comment:
🔧 Please make the min length a constant as it needs to be shared across multiple components and screens and set it to `8`.

Reply:
I was under the impression the 8 was the minimum and acceptable state what's why I did `>7` and not `>8`

I'll change this to

``` typescript
const MIN_PASSWORD_LENGHT = 8
...
passwordField.value.trim().length >= MIN_PASSWORD_LENGHT
```